src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -226,6 +241,19 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
                      BT_DATA(BT_DATA_SVC_DATA16, &serviceData, sizeof(serviceData)),
                      BT_DATA(BT_DATA_NAME_COMPLETE, deviceName, static_cast<uint8_t>(strlen(deviceName))) };
 
+    // Register dynamically CHIPoBLE GATT service
+    if (!mIsChipoBleServiceRegistered)
+    {
+        err = bt_gatt_service_register(&sChipoBleService);
```

Comment:
I think it won't compile after you rebase. Now `CHIP_ERROR` is a class for our platform, not an integer.